examples/advanced/custom-transforms/build.js

```diff
@@ -78,8 +78,8 @@ StyleDictionary.registerTransformGroup({
 
 StyleDictionary.registerTransformGroup({
   name: 'custom/scss',
-  // this is just to show a possibility, if you want to add a few transforms to a pre-defined group
-  // (even so, we suggest to use the previous approach, which is more explicit and clear)
+  // this is to show one possibility, for adding a few transforms to a pre-defined group
```

Comment:
this is to show one possibility for adding a few transforms to a pre-defined group

(removed the comma)